python/ray/train/checkpoint.py

```diff
@@ -308,14 +308,6 @@ def latest_checkpoint_id(self) -> Optional[int]:
 
 
 class TuneCheckpointManager(CheckpointManager):
-    def create_logdir(self, log_dir: Optional[Union[str, Path]]):
-        # Don't create logdir when using with Tune.
-        pass
-
-    def create_run_dir(self):
-        # Don't create run_dir when using with Tune.
-        pass
-
```

Comment:
Any reason for this change? We still want to skip these right?

Reply:
These have actually been deprecated in the superclass I believe, and just haven't been removed in the subclass.